fileparser.py

```diff
@@ -1,29 +1,23 @@
-'''
-fileparser.py - contains basic functions to perform file operations
-'''
-
-class fo:
+class fp:
     def __init__(self, fname):
         self.fname = fname
         self.fname_length = len(fname)
-        self.fname_extension = self.get_extension(fname)
-    
-    def get_extension(self, fname):
-        # use this to introduce methods and attributes
-        i = fname.rfind('.')
-        extension = fname[i+1:]
-        return extension
-    
+        self.extension = self._get_extension()
+        
     def convert_to_list(self):
-        lines = []
```

Comment:
Might be worth adding a brief docstring to the methods, only because I included them in the functions from last week's lesson.

Reply:
Great point, on it